autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -93,6 +95,26 @@ impl UnidentifiedClient {
             })
             .send();
 
+        // This is the first time that the user has connected "today".
+        if flags.user_gm {
+            // Return the number of channels for the user using the internal channel_count.
+            // NOTE: this metric can be approximate since we're sampling to determine the
+            // approximate average of channels per user for business reasons.
+            self.app_state
+                .metrics
+                .incr_with_tags("ua.connection.check")
+                // TODO: These should reflect the actual os and browser, but getting those
+                // values at this point would be very complicated. For now, just use "desktop"
+                .with_tag("os", "desktop")
+                .with_tag("browser", "desktop")
```

Comment:
Ignore this comment if we're killing all this but it could potentially apply to the `.with_tag_value("desktop")` on the histogram:

If we delay emitting this metric until after creating `WebPushClient` a few lines below we can get the parsed `os`/`browser_name`/etc. values from the pub `wpclient.ua_info`